packages/application-generic/src/services/launch-darkly.service.ts

```diff
@@ -20,18 +20,35 @@ export class LaunchDarklyService implements IFeatureFlagsService {
   private client: LDClient;
   public isEnabled: boolean;
 
-  constructor() {
-    Logger.verbose('Launch Darkly service initialized', LOG_CONTEXT);
+  public async initialize(): Promise<void> {
```

Comment:
We move to async initialization to avoid race conditions in tests so Launch Darkly is plugged before executing the tests. Also for deployment of the apps that rely on feature flags through the SDK.